### ISSUE TITLE
SSD Reference for 1.0

### ISSUE THREAD

**Issue by bitfort** [mlcommons/training_policies#385]
Tracking for the SSD reference for 1.0

**Comment by bitfort** [comment]
SWG:

Investigated why some submissions reach a higher average quality score at the 95 epoch. It appears some submissions were using a non-reference backbone and a bias=0 initialization. 

1. We should establish a backbone conversion script for SSD (pytorch to tensorflow). 
2. Establish the expected behavior of the SSD accuracy and have compliance checkers look for potential issues. 
3. Rules should state checkpoints and backbones are provided in top 2 frameworks (currently pytorch and tensorflow) and an automatic conversion script is provided between them.
4. You must use one of these two backbones unless you show the SWG why this isn't possible and propose making another backbone available.

**Comment by pbaumstarck** [comment]
Closing stale issues.
